CesiumGltfReader/src/GltfReader.cpp

```diff
@@ -220,20 +221,21 @@ void postprocess(
   CesiumGltf::Model& model = readGltf.model.value();
 
   if (options.decodeDataUrls) {
-    decodeDataUrls(reader, readGltf, options.clearDecodedDataUrls);
+    decodeDataUrls(reader, readGltf, options);
   }
 
   if (options.decodeEmbeddedImages) {
     CESIUM_TRACE("CesiumGltf::decodeEmbeddedImages");
-    for (Image& image : model.images) {
+    for (CesiumGltf::Image& image : model.images) {
```

Comment:
There's a `using namespace CesiumGltf;` declaration at the top, so qualifying with `CesiumGltf::` shouldn't be necessary. Just for clarity?